src/capwap_http_api_handler.erl

```diff
@@ -181,6 +195,17 @@ handle_request_dp(<<"GET">>, [<<"stats">>], Req,
 handle_request_dp(_, _, Req, State) ->
     {jsx:encode([{error, bad_command}]), Req, State}.
 
+handle_request_metrics(text, Path, Req, State) ->
+    Entries = lists:foldl(fun exo_entry_to_list/2, [],
+                          exometer:find_entries(Path)),
+    Metrics = lists:foldl(fun(M, A) -> maps:get(ioize(M), A) end, Entries,
```

Comment:
I doubt that this will work. Entries is a list and the maps:get won't work on a list.